run-tests.sh

```diff
@@ -68,22 +70,28 @@ else
   echo "No selftests found, disabling"
 fi
 
+if (( $# > 0 )); then
```

Comment:
This is a bit uncommon in Bash, what about `if [ "$#" -ne 0 ]; then` instead?

Reply:
That's what I had at some point in the past, but a colleague asked me to use `(())` instead. Some people say that `[` should be deprecated so :shrug: I'll just keep it as it is.